samcli/lib/pipeline/init/plugins/two_stages_pipeline/postprocessor.py

```diff
@@ -0,0 +1,104 @@
+"""
+The plugin's postprocessor prints information about the created and reused AWS resources and the required permissions
+"""
+import logging
+from typing import Dict, List
+
+import click
+
+from samcli.lib.cookiecutter.processor import Processor
+from .config import PLUGIN_NAME
+from .context import Context as PluginContext
+from .resource import Deployer, Resource
+
+LOG = logging.getLogger(__name__)
+
+
+class Postprocessor(Processor):
+    """
+    prints information about the created and reused AWS resources and the required permissions
+
+    Attributes
+    ----------
+    resources_reused:
```

Comment:
nit: reused sort of suggests that we are reusing our resource, probably provided will be a better word. But no strong preference.